modules/atomic-widgets/props-resolver/transformers/styles/background-image-overlay-transformer.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets\PropsResolver\Transformers\Styles;
+
+use Elementor\Modules\AtomicWidgets\PropsResolver\Transformer_Base;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Background_Image_Overlay_Transformer extends Transformer_Base {
+		public function transform( $value, $key ) {
+
+			if ( ! empty( $value['image-src']['id'] ) ) {
+				$image_src = wp_get_attachment_image_src(
```

Comment:
maybe reduce nesting?
wrap wp_get_attachment_image_src is another function and throw the exception there